python/nav/models/sql/changes/sc.05.15.0001.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE profiles.account_dashboard
+ADD COLUMN is_shared BOOLEAN NOT NULL DEFAULT FALSE;
+
+CREATE TABLE profiles.account_dashboard_subscription (
+    id SERIAL PRIMARY KEY,
+    account_id INT REFERENCES account(id) ON UPDATE CASCADE ON DELETE CASCADE,
+    dashboard_id INT REFERENCES account_dashboard(id) ON UPDATE CASCADE ON DELETE CASCADE,
+    subscribed_at TIMESTAMP NOT NULL DEFAULT NOW(),
```

Comment:
Where does this come from? It's not in the model

Reply:
I am used to adding timestamps to models for debugging/analysis, and thought I might use it for something related to the feature. E.g. a list of current subscribes (and when they subscribed). However, I ended up not using this anywhere in the PRs. It doesn't seem like you are using timestamps in other models, so I can just remove it.